moderatorFrontend/src/main/kotlin/util/LocalizedString.kt

```diff
@@ -24,12 +24,8 @@ object MbLocalizedStringConfig {
     return if (cachedCookieLang == null) {
       //language=JavaScript
       val langFromCookie =
-          js("""document.cookie.replace(/(?:(?:^|.*;\s*)MbLang\s*\=\s*([^;]*).*$)|^.*$/, "$1")""") as String
-      val newLang = if (langFromCookie.isBlank()) {
-        defaultLanguage.langCode
-      } else {
-        langFromCookie
-      }
+        js("""document.cookie.replace(/(?:(?:^|.*;\s*)MbLang\s*\=\s*([^;]*).*$)|^.*$/, "$1")""") as String
+      val newLang = langFromCookie.ifBlank { defaultLanguage.langCode }
```

Comment:
not sure about readability here

Reply:
i was initially also not sure, but intellij recommended. seems we should get along with the new way :D